internal/helm/templatev1beta2/stack.go

```diff
@@ -14,10 +15,31 @@ type StackList struct {
 	Items []Stack `yaml:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
+// TypeMeta is a rewrite of metav1.TypeMeta which doesn't have yaml annotations
```

Comment:
Is this needed for `apiVersion` (instead of `apiversion`) ?

Reply:
exactly

Reply:
I really wonder why the "real" TypeMeta is missing these annotations... We can't seriously be the first to use this type to serialize some `YAML` config...